content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 
-Additionally, the
-link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Board#GovernanceBoard-Security[Jenkins
-Security Officer], with help from members of
-link:https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+CERT+team[Jenkins CERT]
-work with security researchers, plugin and core developers to release security
-fixes and advisories for plugin and core in a timely manner.
+== Security Advisories
 
-// TODO: Once there is a good Handbook chapter it should be prominently linked
-// from here
+See our link:/security/advisories/[archive of past security advisories] for previously published security advisories.
 
+== Security Notifications
 
-== Security Advisories
+We use the +jenkinsci-advisories+ Google group to inform users about upcoming, scheduled security updates:
 
-From time to time, we issue a security advisory to report security problems in
-Jenkins. You can receive notifications for such advisories in one of the
-following ways:
+- link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[+jenkinsci-advisories+ list archive]
+- link:https://feeds.feedburner.com/jenkins-security-advisories[+jenkinsci-advisories+ RSS feed]
 
-. jenkinsci-advisories@googlegroups.com (_read-only_) mailing list (link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[list archive])
-. link:https://feeds.feedburner.com/jenkins-security-advisories[jenkins-security-advisories RSS feed]
+Once a security update has been released, we inform the +jenkinsci-advisories+ and link:http://oss-security.openwall.org/wiki/mailing-lists/oss-security[+oss-security+] mailing lists.
 
-=== Archived Advisories
+We recommend you subscribe to link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[+jenkinsci-advisories+] even if your Jenkins instance is not publicly accessible to receive timely notifications on security updates.
 
-See our link:/security/advisories/[archive of past security advisories] for previously published security advisories.
+== How to Report an Issue
+
+If you find a vulnerability in Jenkins, please report it in the issue tracker under the link:https://issues.jenkins-ci.org/browse/SECURITY[SECURITY project].
+This project is configured in such a way that only the reporter and the security team can see the details.
+By restricting access to this potentially sensitive information we can work on a fix and deliver it before the method of attack becomes well-known.
+
+To show our appreciation for your help, we offer link:https://wiki.jenkins-ci.org/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for your report.
+
+== Jenkins Security Team
+
+The Jenkins Security Team is a group of volunteers lead by the link:https://wiki.jenkins-ci.org/display/JENKINS/Team+Leads[Jenkins Security Officer] who triage and fix security vulnerabilities.
+
+Membership is generally open to anyone with a history of positive contributions to the Jenkins project and subject to approval by the Jenkins Security Officer.
```

Comment:
I see no particular need and it would just be more effort to keep current. We can't even keep https://wiki.jenkins-ci.org/display/JENKINS/Infrastructure+admins up to date.